### PR TITLE
Remove hardcoded cross signing certificate

### PR DESCRIPTION
**Pull request by faern**
Removes the hardcoded CA, that is now outdated and wrong anyway. And instead of just updating it to the new trusted root, make it configurable. This both makes the repository rot less over time, but it also makes it more generic to build with non-Digicert certificates. What do you think?

I have sadly not tried this out. That's not optimal. But actually running this is such a hassle, especially if I have to do it on the build server.

One could argue that the timestamp server should also be configurable then. But in practice that does not give us any benefit right now, it just makes the script harder to call.

### CODE REVIEW

File: MULLVAD_BUILD.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ Configure signing in Microsoft partner portal:
 
 1. Clone/pull updated driver code on trusted build machine.
 1. Launch `Developer Command Prompt for VS 2019`.
-1. Run `build.bat <certificate-sha1-thumbprint>` to build and sign the driver.
+1. Run `build.bat <certificate-sha1-thumbprint>` `<path-to-CA-root-certificate-file>` to build and sign the driver.
```
Comment (dlon) [inline MULLVAD_BUILD.md]: Nit: Use a single pair of backticks: `build.bat <certificate-sha1-thumbprint> <path-to-CA-root-certificate-file>`

File: build.bat [hunk 2]
```diff
@@ -8,7 +8,7 @@ if [%VisualStudioVersion%]==[] (
 if [%1]==[] goto USAGE
```
Comment (dlon) [inline build.bat:8]: This should jump to `USAGE` if `%2` is unset as well. Also, the "usage" text should be updated.

**Review by dlon — CHANGES_REQUESTED** [review]

**Review by dlon — APPROVED** [review]

File: MULLVAD_BUILD.md (hunk 1 above)
Comment (faern) [reply MULLVAD_BUILD.md]: Oops! My mistake.

File: build.bat (hunk 2 above)
Comment (faern) [reply build.bat:8]: Good catch!

**Pull request merged**